src/__integrations__/pm.ts

```diff
@@ -19,28 +18,24 @@ pm.use(bodyParser.json());
 // Use router to keep the express app extensible
 const walletRouter = Router();
 walletRouter.post('/pp-restapi/v3/users/actions/start-session', function (req, res) {
-  StartSessionRequest.decode(req.body).fold(
-    () => res.sendStatus(500),
-    decodedReq => {
-      if (decodedReq.data?.email === 'tooManyRequests@pm.com') {
-        return res.sendStatus(429);
-      } else if (decodedReq.data && Object.keys(decodedReq.data).length === 0) {
-        return res.sendStatus(422);
-      } else if (!Object.prototype.hasOwnProperty.call(decodedReq.data, 'email')) {
-        return res.sendStatus(500);
-      } else {
-        return res.json({
-          data: {
-            sessionToken: myFake.random.alphaNumeric(128),
-            user: {
-              email: decodedReq.data?.email,
-              status: 'ANONYMOUS',
-            },
-          },
-        });
-      }
-    },
-  );
+  const decodedReq = req.body;
```

Comment:
@infantesimone Don't you think it's useful to use the autogenerated decoder?

Reply:
I deleted it because it don't allow to differentiate the two cases:

- body `{ }` => 500
- body `{ data : {} }` => 422

but....is the test right?